README.md

```diff
@@ -14,6 +14,7 @@ Interested in learning more? Read on!
 
 Want to just get started now? Check out the [instructions](#getting-started).
 
+See a practical tutorial on how to create a builder image [here](exmaples/TUTORIAL.md)
```

Comment:
I'd move this sentence to the end of [Anatomy of a builder image](https://github.com/openshift/source-to-image/#anatomy-of-a-builder-image) section. Not everyone who's interested in s2i will be interested in creating builder images. Besides that section provides good intro to your tutorial.